catalyst_voices/packages/catalyst_voices_brands/lib/src/themes/catalyst.dart

```diff
@@ -265,6 +268,27 @@ TextTheme _buildTextTheme(VoicesColorScheme voicesColorScheme) {
   );
 }
 
+/// Most of buttons share same configuration and differ in just a
+/// few properties that's why we're extracting what's shared.
+///
+/// ButtonStyle returned by this function is not final and is meant to
+/// be served as further adjustment via .copyWith or .merge
+ButtonStyle _buildBaseButtonStyle(TextTheme textTheme) {
+  return ButtonStyle(
+    textStyle: WidgetStatePropertyAll(textTheme.labelLarge),
+    minimumSize: const WidgetStatePropertyAll(Size(85, 40)),
+    iconSize: const WidgetStatePropertyAll(16),
+    shape: const WidgetStatePropertyAll(StadiumBorder()),
+    padding: const WidgetStatePropertyAll(
+      EdgeInsets.symmetric(horizontal: 16, vertical: 10),
+    ),
+  );
+}
+
+/// Note:
+///
+/// If we're going to introduce other themes then catalyst this method
+/// should be extracted
```

Comment:
Can you please review all documentation sentences you added in this PR and add `.` at the end of each sentence. I know it's rather boring but we should keep once standard.